.travis.yml

```diff
@@ -1,5 +1,3 @@
-sudo: false
```

Comment:
👍 

`sudo: false` is deprecated for over a year now. Thanks for the clean up!

Reference:
https://docs.travis-ci.com/user/reference/trusty/#container-based-infrastructure
https://blog.travis-ci.com/2018-10-04-combining-linux-infrastructures
https://blog.travis-ci.com/2018-11-19-required-linux-infrastructure-migration#timeline---its-happening-fast

Reply:
extracted to #146